var/httpd/htdocs/js/Core.Agent.DynamicFieldDBSearch.js

```diff
@@ -276,7 +284,7 @@ Core.Agent.DynamicFieldDBSearch = (function(TargetNS) {
                         // run the response function to hide the request animation
                         Response({});
                     }
-
+                    
```

Comment:
Please remove the blanks. Usually this is done by the CodePolicy, but we have to improve it, to run with js in a meaningful way, too...